canonical-web-and-design/jaas.ai
# Remove the tutorials section

## Done
The tutorials section on jaas.ai is out of date and needed to be repaired. But have decided to decommission the jaas.ai based tutorials section and redirect to juju.is/tutorial

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Check all the pages work as before
- Go to /tutorials
- See it redirects to juju.is/tutorials
